Sources/SFSymbol/UIImage+SFSymbol.swift

```diff
@@ -25,11 +25,11 @@ import UIKit
 
 @available(iOS 13.0, *)
 public extension UIImage {
-    convenience init?(symbol: SFSymbol) {
-        self.init(systemName: symbol.rawValue)
+    convenience init(symbol: SFSymbol) {
+        self.init(systemName: symbol.rawValue)!
```

Comment:
@Rspoon3 Thanks for the PR, I understand what you mean but I think that things like force unwrapping should be left as a choice to the user of the library. Hmm... Still thinking of how else this can be done.